scrapper.py

```diff
@@ -1,6 +1,28 @@
 """
 Crawler implementation
 """
+import json
+import os
+import time
+import random
+import shutil
+import logging.config
```

Comment:
1. imports in alphabetical order
2. imports in following order: Python standard library, empty line, external libraries, empty line, local modules

Reply:
addressed this issue